dash/backend/src/modules/image/dao/image.dao.ts

```diff
@@ -178,12 +178,12 @@ export class ImageDao {
         });
     }
 
-    async countImage(searchClause: any): Promise<any> {
+    async countImage(searchClause: any): Promise<string> {  // this was always returning a string. now it's typed appropriately.
```

Comment:
Wouldn't it make more sense to cast this to a number rather than having a count variable be a string?

I checked, and this function is hooked up to an endpoint, although the frontend does not call that endpoint, it does expect a number. I'm guessing that this worked by coincidence because the string was just being passed to a paginator which would cast the type autoamtically.

Reply:
It would make sense, but I didn't want to change the functionality. Like the comment indicates, it was always returning a string and not a number, so I didn't want to change the functional type.

Reply:
This function was not being used before the changes you made, so it would be the perfect opportunity to fix the typing.
Why make a comment explaining that the code is weird rather than make it better when there are no regressions?

Reply:
Because typically you don't want to change what's functionally being returned via an API just in case someone or something is depending on it being odd.
I can change it though.